.github/workflows/generate.yml

```diff
@@ -0,0 +1,54 @@
+name: Regenerate Asset Data and PR
+
+on:
+  # Allows manual triggering if we ever want to
+  workflow_dispatch:
+  schedule:
+  - cron: '0 9 * * 1,4' # Twice a week - 9AM UTC on Monday and Thursday
+
+jobs:
+  regenerate-and-create-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout develop
+        uses: actions/checkout@v3
+        with:
+          ref: develop
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+
+      - name: Yarn Install
+        run: yarn install --immutable
+
+      - name: Setup env
+        run: yarn env dev
+
+      - name: Build Packages
+        run: yarn build:packages
+
+      - name: Generate asset data
+        env:
+          ZERION_API_KEY: ${{ secrets.ZERION_API_KEY }}
+        run: yarn run generate:asset-data
+
+      - name: Create new feature branch
+        run: git checkout -B feat_regenerate_asset_data
+
+      - name: Commit changes
+        run: |
+          git checkout -B feat_regenerate_asset_data
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add -A
+          git diff --staged --quiet || git commit -m "feat: regenerate asset data"
```

Comment:
might want to add a date to the branch and pr title to make these unique and searchable

Reply:
Good idea! I'd probably keep the branch name static to keep things simpler and not programmatic branch-name wise, since we could use the diff/PR to refer back to it either way.
PR title on the other hand would absolutely help with search 👀 
cc21898f27 and 78452721f1

Reply:
only issue would be if we didn't merge within the cron time period and stepped on previous branch state (basically ending up with two prs with the same changeset). guess we could just close one, but would lose the incremental diff

Reply:
Oh absolutely - the idea here is optimism and those get merged quickly after being opened after a quick sanity test. Not super concerned about the case of stale ones!